lib/koala/errors.rb

```diff
@@ -82,6 +88,16 @@ def initialize(http_status, response_body, error_info = nil)
 
         super(message)
       end
+
+      private
+
+      # refs: https://developers.facebook.com/docs/graph-api/overview/rate-limiting/#headers
+      # NOTE: The header will contain a JSON-formatted string that describes current application rate limit usage.
+      def json_formatted(string)
+        JSON.parse(string || '')
+      rescue JSON::ParserError
+        nil
```

Comment:
Could you log the error with Koala logger and error level, including the error class + message and the raw value that was returned in the header.
This way there is no supprise to anyone and if someone has to report a bug they can do it with the proper logs

Reply:
Okay, I added logging for the error and contained the raw error info.